exercises copy/A-setup-ide/README.md

```diff
@@ -0,0 +1,13 @@
+There are some tools that will help you to write code. One of these, [Prettier](https://prettier.io/), formats your code, making it easier for you and others to read.
```

Comment:
it looks like you've copied the whole `exercises` folder and committed it, hence it shows up in the pull requests as new files. Be careful to only commit your changes so it's easier for the pull request to be reviewed.